### DISCUSSION TITLE
Getting the x,y coordinates from a model

### DISCUSSION THREAD

**Discussion by semcsherry** [ultralytics/ultralytics#3492] [Q&A] [answered]
I'm very new to yolov8 and python.   Would someone be able to point me to a good example for getting the x and y coordinates of the keypoints of the model (when using a pose model) using Python?

Thank you!

**Comment by glenn-jocher** [comment, marked as answer]
@semcsherry hello! Welcome to the YOLOv8 community. 😊

To obtain the x, y coordinates of keypoints when using a pose model with YOLOv8, you'll want to follow these general steps in Python:

1. Load your trained pose estimation model.
2. Run inference on your image or video frame.
3. Access the keypoints from the inference results.

The keypoints are typically returned in the output as a list of coordinates, each corresponding to a specific part of the pose detected. You can iterate through these results to get the x, y coordinates for each keypoint.

For detailed guidance on how to perform each of these steps, please refer to the Pose/Keypoint Estimation section in our documentation. The docs will provide you with a clear understanding of how to work with pose models and extract the information you need.

If you have any more questions or need further assistance, feel free to ask. The community and the Ultralytics team are here to help. Good luck with your YOLOv8 journey! 🚀
